### ISSUE TITLE
Contributing instructions

### ISSUE THREAD

**Issue by dorner** [hotwired/stimulus#325]
There seems to be some process to build and package Stimulus which I'm not familiar with. I'm trying to contribute a PR but I can't figure out how to a) build the package and b) link it into an existing project so I can test it. There seems to be a core repo, then a sub-package which is *also* called `stimulus`, then a number of sub-packages which are *not* published to NPM separately but bundled together somehow into the `stimulus` package, both in a `node_modules` directory and a single rolled-up file. Lerna seems to have something to do with it, but there are no actual instructions anywhere.

Long story short, can someone please tell me what I have to do to get this working? :)

**Comment by adrienpoly** [comment]
What I did was to use the `yarn link` command to create a [local symlink to the build](https://classic.yarnpkg.com/en/docs/cli/link/) 

```bash
# create the simlink
cd packages/stimulus
yarn link 

# add the simlink to you project
cd yourproject
yarn link "stimulus"
```

Then you can use either `yarn dev` to continuously build at every change or `yarn build`

**Comment by dorner** [comment]
pretty sure I tried this and it didn't work because the changes I made in e.g. `@stimulus/core` didn't make its way into the linked directory.

Also, my linked project uses npm, not yarn... not sure how that's going to square. :(

**Comment by adrienpoly** [comment]
npm has the same tool https://docs.npmjs.com/cli/link

last time I tried (with yarn) I could get the change in my test project.

be sure that you make the link in the correct folder `packages/stimulus`

**Comment by dorner** [comment]
I can link the two projects together, but if I make a change to one of the `.ts` files in `@stimulus/core` and then run `yarn build` in either the root directory or the `stimulus` package, it doesn't seem to update `stimulus.umd.js` nor what's in `node_modules`, which is what seems to actually be included in the other project via npm.

**Comment by adrienpoly** [comment]
what should be updated on yarn build  is the `packages/@stimulus/core/dist` directory (and other but the most important is this one)

also from your project if you inspect you node_modules directory you should have a symlink like this (with npm I suppose)

![Screen Shot 2020-08-25 at 8 38 44 PM](https://user-images.githubusercontent.com/7847244/91214462-639f4a80-e713-11ea-9e92-5dadc3ae7151.jpg)

**Comment by dhh** [comment]
We added a CONTRIBUTING.md guide to Turbo. Feel free to explore a PR that adds the same to Stimulus: https://github.com/hotwired/turbo/pull/194
